lib/cylc/task_remote_mgr.py

```diff
@@ -272,7 +271,7 @@ def remote_tidy(self):
                 if proc.poll() is None:
                     continue
                 del procs[(host, owner)]
-                out, err = proc.communicate()
+                out, err = (f.decode() for f in proc.communicate())
                 if proc.wait():
                     LOG.warning(TaskRemoteMgmtError(
                         TaskRemoteMgmtError.MSG_TIDY,
```

Comment:
Hummm, it appears to have a but on line 279. Not touched in this pull request, but I wonder if we can address it here... it's using `proc.ret_code`, but I just had a look at `subprocess.py` in my Anaconda Python installation, and it has only `.returncode`. Line 291 has actually a very similar code, but using `returncode`. If out of scope, I can create a separate issue for this one.

Reply:
Might as well fix it here.